fe1-web/src/features/rollCall/screens/__tests__/RollCallScanner.test.tsx

```diff
@@ -231,4 +232,20 @@ describe('RollCallOpened', () => {
       ],
     });
   });
+
+  it('shows the correct number of attendees', async () => {
+    const mockAttendeePopTokens = [mockPopToken.publicKey.valueOf()];
+
+    const { getByText } = renderRollCallOpened(mockAttendeePopTokens);
+
+    expect(getByText(`${STRINGS.roll_call_scan_counter}: 0`)).toBeTruthy();
```

Comment:
Indeed it would look better, but I am not too sure on how to do it, would it be something like this?

```Typescript
    expect(getByTestId('roll_call_attendee_counter_text')).toHaveTextContent(`${STRINGS.roll_call_scan_counter}: 0`);
```

It seems to be able to do this we need to add an import (found this info [here](https://github.com/testing-library/react-testing-library/issues/379)):
`import "@testing-library/jest-dom/extend-expect";`
and this import currently is not recognized within our project 

Also` getByTestId('roll_call_attendee_counter_text').textContent` does not work (also need an import not recognized by our project), do you know by any chance an another way of doing the test with the testID? (I couldn't find it in our project, the tests with getByTestId were all about pressing buttons)

Reply:
I do not think it is better, since what we really want to check is that the specific text is indeed displayed. If we want to check that it is displayed at the right position, then we would need a snapshot.

The main addition of Xelowak's solution is to check that the text is written in a specific object. Since we do not have the position of the object in the screen, i do not think it is useful compared to the current one.

Reply:
You're right, I think your point is well argued! Feel free to merge this branch then.

Reply:
Your point is well argued @MeKHell, but I guess you also want to test that the text is correctly displayed in the correct component. As I said, I don't want t to be blocking anyway so feel free to merge if you think it's ok like this.

Feel free to add testing library if you need it, especially if what it does is not currently supported.
I found something similar to your test on StackOverFlow: https://stackoverflow.com/questions/58976251/checking-text-appears-inside-an-element-using-react-testing-library